### ISSUE TITLE
Installing toolbox through bootstrap script fails with Swift 3.0 release.

### ISSUE THREAD

**Issue by alokjha** [vapor/toolbox#93]
```
Aloks-MacBook-Pro:~ alok$ which swift
/usr/bin/swift
Aloks-MacBook-Pro:~ alok$ swift --version
Apple Swift version 3.0 (swiftlang-800.0.46.2 clang-800.0.38)
Target: x86_64-apple-macosx10.9
```

```
Aloks-MacBook-Pro:~ alok$ curl -sL toolbox.vapor.sh | bash
✅  Compatible
Downloading...
Compiling...
Installing...
Install failed, trying sudo
Password:
Error: Installation failed.
Use vapor --help and vapor <command> --help to learn more.
```

**Comment by jimmysmith95** [comment]
I think this may be related to a problem I had earlier today. Do you have a /usr/local/bin directory? If not, you could try creating that first and then running it. (Before I think homebrew was required in the installation process so everyone had this directory by the time they got to installing toolbox)

**Comment by jimmysmith95** [comment]
I also have a PR to create that directory if it doesn't already exist here: https://github.com/vapor/toolbox/pull/97

**Comment by tannernelson** [comment]
Fixed in 1.0.3
